06-building-a-program-start-to-finish/writing-tests/output/spec.js

```diff
@@ -0,0 +1,122 @@
+/* My Test Setups */
+/* global describe it expect */
+import myBook from './books.json';
+import emptyBook from './emptyBook.json';
+import invalidKeys from './Invalidkeys.json';
+
```

Comment:
A base filename should exactly match the name of its default export. [See link](https://github.com/airbnb/javascript#naming--filename-matches-export)